_docs/example-collections.md

```diff
@@ -547,12 +547,12 @@ JSON_START
         // recipient details from the previous section
     }],
     "payin_methods": [{
-        "type": "GHS::Mobile",
+        "type": "GHS::Mobile", 
+        "ux_flow": "ussd_popup",
         "in_details": {
-            "phone_number": "+256301234567",
-            "send_instructions": true
-        },
-        "provider": "interpay"
+            "phone_number": "+2339999999", // In international format
+            "mobile_provider": "vodafone" // Optional. One of 'airtel', 'tigo', 'mtn', 'vodafone'
```

Comment:
```suggestion
            "mobile_provider": "vodafone" // Mandatory. One of 'airtel', 'tigo', 'mtn', 'vodafone'
```
We're now considering this mandatory from the documentation perspective. Likewise for the other cases it is used below.